create/how_to/updating.md

```diff
@@ -0,0 +1,231 @@
+
+Now that you’ve got your dump, you just need to [install the latest version of MeiliSearch](/learn/getting_started/installation.md#download-and-launch) and import the dump at launch, like so:
+
+```bash
+# Launch MeiliSearch and import the specified dump file
+./meilisearch --import-dump /dumps/your_dump_file.dump
+```
+
+> Importing a dump is the same process as indexing documents. This can take time and cause a spike in RAM usage. To save RAM, use a [smaller batch size](/reference/features/configuration.md#dump-batch-size).
+
+Finally, don’t forget to set `displayedAttributes` back to its previous value, if necessary.
+
+Congratulations! You have successfully migrated your MeiliSearch database to the latest version! 🎉
+
+## Updating from v0.14.0 or below
+
+Since these versions predate the [dumps feature](/reference/features/dumps.md), the best solution is to export your documents and your [index settings](/reference/features/settings.md) as `.JSON` files.
+
+If you don’t need to preserve index settings, skip directly to [step 2](#step-2-set-all-fields-as-displayed-attributes).
+
+### Step 1: Save your settings
+
+First, use the [get settings endpoint](/reference/api/settings.md#get-settings) to retrieve the [settings](/reference/features/settings.md) of any indexes you want to preserve, and save them to a file using the method you prefer.
+
+```bash
+# the -o option saves the output as a local file
+curl -X GET \
+  'http://127.0.0.1:7700/indexes/:index_uid/settings' -o mysettings.json
+```
+
+Repeat this process for all indexes you wish to migrate.
+
+**It is important to save your settings before exporting documents**, since the next step may require you to modify the settings.
+
+### Step 2: Set all fields as displayed attributes
+
+To prevent data loss, all fields must be set as [displayed](/reference/features/field_properties.md#displayed-fields).
+
+By default, all fields are added to the displayed attributes list. Still, it's a good idea to verify this before creating a dump. You can do so by using the [get displayed attributes endpoint](/reference/api/displayed_attributes.md#get-displayed-attributes):
+
+```bash
+curl -X GET \
+  'http://127.0.0.1:7700/indexes/:index_uid/settings/displayed-attributes'
+```
+
+If the returned value is `["*"]`, you can move on to the [next step](#step-2-create-the-dump). If not, then you need to use the [reset displayed-attributes endpoint](/reference/api/displayed_attributes.md#reset-displayed-attributes).
+
+```bash
+curl -X DELETE \
+  'http://127.0.0.1:7700/indexes/:index_uid/settings/displayed-attributes'
+```
+
+Now that all fields are displayed, you can proceed to the next step.
+
+### Step 3: Save your documents
+
+Use the [get documents endpoint](/reference/api/documents.md#get-documents) to retrieve your documents and save them using the method you prefer. Make sure to set the `limit` parameter so that, if you have `n` documents, `limit ≥ n`. Otherwise, you risk data loss.
```

Comment:
can we let them know what is the limit, or tell them it's explained in the link we just added, or remove that part since it's explained in the API reference?

Reply:
Don't remove this part, people will lose their documents if they don't raise the limit.